meilisearch-release-changelog/v1.7.0.md

```diff
@@ -0,0 +1,223 @@
+
+# New features and improvements 🔥
+
+## Improve AI with Meilisearch (experimental feature)
+
+🗣️ AI work is still experimental and we need your help to improve it! Share your thoughts and feedback on this [GitHub discussion](https://github.com/orgs/meilisearch/discussions/677).
+
+To use it, you need to enable `vectorSearch` through [the `/experimental-features` route](https://www.meilisearch.com/docs/reference/api/experimental_features).
+
+💡 More documentation about AI search with Meilisearch [here](https://www.meilisearch.com/docs/learn/experimental/vector_search#vector-search-with-auto-embeddings).
+
+### Add new OpenAI embedding models & abbility to override their models dimensions
+
+When using `OpenAi` as `source` in your `embedders` index settings (an example [here](https://www.meilisearch.com/docs/learn/experimental/vector_search#generate-auto-embeddings-with-openai)), you can now specify two new models:
+- `text-embedding-3-small` with a default dimension of 1536.
+- `text-embedding-3-large` with a default dimension of 3072.
+
+The new models:
+- are cheaper
+- produce more relevant results in standardized tests
+- allow to set up the dimensions of the embeddings to control the trade-off between accuracy and performance (including storage)
+
+It means that it is now possible to pass the dimensions field when using the `OpenAi` source. This was previously only available for the `userProvided` source in the previous releases.
+
+There are some rules, though, we detail with these examples:
+
+```json
+"embedders": {
+  "large": {
+    "source": "openAi",
+    "model": "text-embedding-3-large",
+    "dimensions": 512 // must be >0, must be <= 3072 for "text-embedding-3-large"
+  },
+  "small": {
+    "source": "openAi",
+    "model": "text-embedding-3-small",
+    "dimensions": 1024 // must be >0, must be <= 1536 for "text-embedding-3-small"
+  },
+  "legacy": {
+    "source": "openAi",
+    "model": "text-embedding-ada-002",
+    "dimensions": 1536 // must =1536  for "text-embedding-ada-002"
+  },
+  "omitted_dimensions": { // uses the default dimension
+    "source": "openAi",
+    "model": "text-embedding-ada-002",
+  }
+}
+```
+
+Done in #4375 by Gosti.
+
+### Add GPU support to compute embeddings
+
+Enabling the CUDA feature allows using an available GPU to compute embeddings with a `huggingFace` embedder.
+On an AWS Graviton 2, this yields a x3 - x5 improvement on indexing time.
+
+👇 How to enable GPU support through CUDA for HuggingFace embedding generation:
+
+**Prerequisites**
+
+- Linux distribution with a compatible CUDA version
+- NVidia GPU with CUDA support
+- A recent Rust compiler to compile Meilisearch from source
+
+**Steps**
+
+1. Follow [the guide](https://huggingface.github.io/candle/guide/installation.html) to install the CUDA dependencies
+2. Clone Meilisearch: `git clone https://github.com/meilisearch/meilisearch.git`
+3. Compile Meilisearch with the `cuda` feature: `cargo build --release --features cuda`
```

Comment:
By adding the `--package` flag you force cargo to only compile the meilisearch binary, nothing more.

```suggestion
3. Compile Meilisearch with the `cuda` feature: `cargo build --release --package meilisearch --features cuda`
```